misc/config_definitions.json

```diff
@@ -4943,6 +4943,16 @@
                 "value.*": "exists:devices,type"
             }
         },
+        "oxidized.ignore_device_groups": {
```

Comment:
Saving group names could be problematic, users can rename groups.
It would be best to save group_ids.   (this also simplifies the filtering code)